### DISCUSSION TITLE
如何优雅的给antd debug

### DISCUSSION THREAD

**Discussion by sunsunmonkey** [ant-design/ant-design#46225] [Q&A] [answered]
我想要给antd改点bug，目前没找到太好的方式来给antd打断点调试🙃

**Comment by Wxh16144** [comment]
antd 源码的话，我一般直接 console 调试

**Comment by vaynevayne** [comment]
希望有朝一日, 可以实现链接github 上,  我 fork 了的项目

**Comment by yoyo837** [comment, marked as answer]
项目启动，浏览器打断点，和自己项目一样的

**Reply by sunsunmonkey** [reply]
刚刚试了下确实，就直接在本地启动ant-design的文档，终端调试和那个vscode launch好像都可以，感谢🥳
